cdap-ui/app/features/apps/templates/detail.html

```diff
@@ -53,13 +65,27 @@
     </div>
   </div>
   <div class="cdap-subnav-end"></div>
-  <ul class="nav nav-tabs slanted-tabs" role="tablist">
+  <ul class="nav nav-tabs slanted-tabs" role="tablist" ng-if="DetailController.isHydrator">
 
     <li role="presentation" ui-sref-active="active">
       <a ui-sref="apps.detail.overview.status" ui-sref-opts="{reload: true}" role="tab">Status</a>
     </li>
```

Comment:
Is it possible to put this <li> block to be inside the <ul> from line 83-92, and simply put the ng-if isHydrator on the <li> itself?